lib/kubernetes-deploy/restart_task.rb

```diff
@@ -29,14 +29,15 @@ def initialize(context:, namespace:, logger: nil, max_watch_seconds: nil)
       @max_watch_seconds = max_watch_seconds
     end
 
-    def perform(*args)
+    def run(*args)
       perform!(*args)
       true
     rescue FatalDeploymentError
       false
     end
+    alias_method :perform, :run
 
-    def perform!(deployments_names = nil, selector: nil)
+    def run!(deployments_names = nil, selector: nil, verify_result: true)
```

Comment:
Will we be moving `deployment_names` to an initialization parameter at some point? It seems strange that we wait to declare it until `run!`.

Reply:
I think once we break the public API out next deploy has to be 1.0. I've been trying to hold off on that a big longer since there are some bug fixes that might be nice to release first.